package.json

```
@@ -41,15 +42,17 @@
     "tabWidth": 2
   },
   "devDependencies": {
+    "@babel/core": "^7.0.0",
```

```suggestion
    "@babel/core": "^7.1.2",
```
Maybe update to latest versions?

---

Good call. I used https://github.com/babel/babel-upgrade but that tool seems to have `7.0.0` hardcoded :) Will do.